hosted-cluster-config-operator/controllers/resources/registry/registry.go

```diff
@@ -26,7 +27,7 @@ func ReconcileRegistryConfig(cfg *imageregistryv1.Config, isPlatformNone bool) {
 	cfg.Spec.Requests.Write.MaxRunning = 0
 	cfg.Spec.Requests.Write.MaxWaitInQueue.Reset()
 
-	if isPlatformNone {
+	if platformType == hyperv1.NonePlatform || platformType == hyperv1.KubevirtPlatform {
 		cfg.Spec.Storage = imageregistryv1.ImageRegistryConfigStorage{EmptyDir: &imageregistryv1.ImageRegistryConfigStorageEmptyDir{}}
 	} else {
 		cfg.Spec.Storage.EmptyDir = nil
```

Comment:
what happens if we don't set this type to `EmptyDir` and leave EmptyDir == nil for kubevirt platform?